packages/generators/src/init-generator.ts

```diff
@@ -74,12 +75,13 @@ export default class InitGenerator extends CustomGenerator {
 
         this.usingDefaults = true;
 
-        process.stdout.write(
+        logger.log(
             `\n${logSymbols.info}${blue(' INFO ')} ` +
                 'For more information and a detailed description of each question, have a look at: ' +
-                `${bold(green('https://github.com/webpack/webpack-cli/blob/master/INIT.md'))}\n`,
+                `${bold(green('https://github.com/webpack/webpack-cli/blob/master/INIT.md'))}`,
         );
-        process.stdout.write(`${logSymbols.info}${blue(' INFO ')} ` + 'Alternatively, run "webpack(-cli) --help" for usage info\n\n');
+        logger.log(`${logSymbols.info}${blue(' INFO ')} ` + 'Alternatively, run "webpack(-cli) --help" for usage info');
+        logger.raw();
```

Comment:
Let's avoid it, if we want empty newline, I would suggest to put `\n` in previously `log` - less code and better readable (related to whole PR)